setupTests.ts

```diff
@@ -20,17 +19,17 @@ const removeAllRulesBut = (spectral: Spectral, ruleName: string) => {
 
   const rawRule = asyncApiRules[ruleName];
 
-  const patchedRule: IRule = Object.assign(rule1, {
```

Comment:
I thought we wanted to try and avoid casting, but this removes a type declaration and introduces a cast.